src/components/TextAvatar/index.tsx

```
@@ -1,14 +1,20 @@
 import { Avatar, Tooltip } from '@mui/material';
 
-function contrastBgColor(bgColor, lightColor, darkColor) {
+interface TextAvatarProps {
+  uid: string;
+  displayName: string;
+  size?: 'small' | 'medium';
+}
+
+function contrastBgColor(bgColor: string, lightColor: string, darkColor: string): string {
   const color = bgColor.charAt(0) === '#' ? bgColor.substring(1, 7) : bgColor;
   const r = parseInt(color.substring(0, 2), 16); // hexToR
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)